app/controllers/job_postings_controller.rb

```diff
@@ -16,6 +18,8 @@ def create
       redirect_to root_path, notice: t(".success")
     else
       flash[:alert] = t(".failure")
+      @experience_levels = ExperienceLevel.all
+      @job_types = JobType.where(status: :active)
```

Comment:
The `enum` method create scopes so we can filter easily: try using `JobType.active`  instead of writing a `where` query.